README.md

```diff
@@ -25,10 +31,28 @@ These dependencies can be installed using the [Homebrew](https://brew.sh/) packa
 * MySQL `$ brew install mysql`
 
 ### Installation
-You will need to clone this repository.
-  1. Clone this repository. `$ git clone https://github.com/team-quizki/quizki-backend.git`
-  2. Change to the quizki-backend directory. `$ cd quizki-backend`
-  3. Configure a remote for your fork. [GitHub Help](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2)
+You will need to fork the [team-quizki/quizki-backend repository](https://github.com/team-quizki/quizki-backend), then clone your new fork to your local computer, and configure the cloned copy on your computer. You can follow these [instruction on Github](https://help.github.com/articles/fork-a-repo/). To summarize this instruction: 
+
+- On GitHub fork the [team-quizki/quizki-backend](https://github.com/team-quizki/quizki-backend) repository.
+
+- Using Git Bash or Git CMD clone your newly forked repository to your local computer:
+    `$ git clone https://github.com/<<UseYourGitHubHandleHere>>/quizki-backend.git`
+
+- Change to the quizki-backend directory. 
+    `$ cd quizki-backend`
+    
+- Configure your local quizki-backend to use remote upstream. 
+    `$ git remote add upstream hhttps://github.com/team-quizki/quizki-backend`  
+    
+- Verify upstream set correctly
```

Comment:
Can we, in a few words, be more specific about what "set correctly" means? IE, what should the user expect to see?

Reply:
Johnathan,

I added example output for the 'git remote -v' 

Hopefully that will work....

Deborah

Reply:
Johnathan ( @haxwell ),

Previously, I updated  the text for showing a sample output for "git remote -v"

Today, I created the \bin at the root, moved the popq2db.bat to .\bin\popq2db.bat, and removed the popq2db.bat from the .gitignore. 

Let me know if you would like to see other changes for the quziki-backend README.md file. If no changes, then please merge the revised README.md file to dev (and perhaps master since it is also wrong.)

Deborah